mantidproject/mslice
# Enable powder(Bragg) peaks on cut plots

**Description of work**

Powder(Bragg) peaks are enabled on cut plots in addition to slice plots. 
Overplot interface is created taking material from powder peaks implementation for slice plots.

**To test:**

Create a slice plot
Make sure Information over-plots on slice plots work without any problems (The existing feature is not corrupted)

Create a cut plot
Then check that powder(Bragg) peaks (in Information) can be invoked on a cut plot


Fixes #602 .
![PowderPeaks](https://user-images.githubusercontent.com/52707387/125275473-f7b3d980-e306-11eb-80d7-79df9dad6f36.jpg)
